Header header

int8 angular_action
int8 linear_action
bool revert